.github/workflows/push-to-crowdin.yml

```diff
@@ -0,0 +1,38 @@
+name: Push files to Crowdin
+
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  docs:
+    name: Push files to Crowdin
+    if: github.repository == 'wailsapp/wails'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Verify Changed files
+        uses: tj-actions/verify-changed-files@v11.1
+        id: verify-changed-files
+        with:
+          files: |
+            website/**/*.mdx
+            website/**/*.md
+            website/**/*.json
+
+      - name: Set node
+        if: steps.verify-changed-files.outputs.files_changed != 'true'
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Push files
+        env:
+          GITHUB_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
```

Comment:
I'm trying to understand what this is for. How does the Crowdin cli read this in?

Reply:
Oh sorry, the key should be: `CROWDIN_PERSONAL_TOKEN`, my mistake, I will update this branch later.